package.json

```diff
@@ -117,9 +119,7 @@
     "react-dom": "^17.0.0",
     "rollup": "^2.32.0",
     "rollup-plugin-babel": "^4.4.0",
```

Comment:
Would you please try https://www.npmjs.com/package/@rollup/plugin-babel too?

Reply:
I thought the same while going to bed last night. I'll try it later today.